integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -39,6 +40,8 @@ COPY --from=psoc6 /opt/Tools/ModusToolbox /opt/Tools/ModusToolbox
 
 COPY --from=telink /opt/telink/zephyrproject /opt/telink/zephyrproject
 COPY --from=telink /opt/telink/zephyr-sdk-0.17.0 /opt/telink/zephyr-sdk-0.17.0
+COPY --from=telink-zephyr_3_3 /opt/telink/zephyrproject /opt/telink/zephyrproject
+COPY --from=telink-zephyr_3_3 /opt/telink/zephyr-sdk-0.17.0 /opt/telink/zephyr-sdk-0.17.0
```

Comment:
is this not the same as line 41/42? How do we merge these?

Maybe paths should be different in both images

Reply:
> is this not the same as line 41/42? How do we merge these?
> 
> Maybe paths should be different in both images

paths of zephyr-sdk and zephyr are the same in both Dockers (versions)
zephyr-sdk-0.17.0 - is fully same for both versions
zephyrproject - is different (but with same path) with different versions of Zephyr
Generally I'm not sure if we need to add this lines, but I see `COPY --from=telink` which mean image_name and as a result I made a copy for other image.

Reply:
I understood mistake and will rework this part

Reply:
@andy31415 
The best approach is to avoid including the old Zephyr 3.3 version in the VSCode image (I removed in last commit change).
This version will be used for about one more year by some customers, but Telink and most customers typically use a local Zephyr setup instead of Docker or VSCode builds.
It's better to maintain consistent paths in both Telink Docker images for both Zephyr versions.
We need this Docker images for GitHub CI builds only.